theano/gpuarray/tests/test_subtensor.py

```diff
@@ -76,6 +77,48 @@ def shared(x, **kwargs):
         assert self.sub == GpuSubtensor
 
 
+def test_advinc_subtensor():
```

Comment:
Why this test? Why it isn't included in test_subtensor.TestAdvancedSubtensor that is reused by G_advancedsubtensor? Should we move them to test_subtensor.TestAdvancedSubtensor to be sure that the CPU also handle well those cases?